client/components/login/LoginButtons/LoginButtons.module.scss

```diff
@@ -14,7 +14,7 @@
 }
 
 .register-mode {
-  button:first-of-type {
+  button:first-of-type:not(.forgot-password) {
```

Comment:
This works, but I think a better fix is to add classes on the Login and Register buttons, and then here only transform those.

This way, other buttons are unaffected.

Reply:
So, if in the future we keep adding buttons to this UI, we won't need to make any changes.